articles/portal/ptl-ref-portal-api.md

```diff
@@ -827,6 +833,9 @@ To specify how many vOrgs to display per page of results, set the `per_page` URL
 
 Returns information about the VDCs, vApps and VMs associated with the specified compute service (vOrg).
 
+> [!NOTE]
+> While the billing information returned by this endpoint is retained for backward compatibility it is no longer accurate. For accurate billing information, use [*GET /api/billing/billing-csv*](#get-apibillingbilling-csv).
```

Comment:
is it better to say "no longer up to date" rather than "no longer accurate"?

Reply:
Amended slightly